CMakeLists.txt

```diff
@@ -111,6 +111,12 @@ endif()
 option(OCIO_USE_SSE "Specify whether to enable SSE CPU performance optimizations" ON)
 option(OCIO_INLINES_HIDDEN "Specify whether to build with -fvisibility-inlines-hidden" ${UNIX})
 
+###############################################################################
+# Supplemental builtins
+
+# Add supplemental built-in transformations such as camera related one.
+option(OCIO_ADD_EXTRA_BUILTINS "Specify whether to add supplemental built-in transforms" ON)
```

Comment:
What are the benefits of making some transforms optional? This seems like it could be problematic in terms of config compatibility.

Reply:
When we presented the feature at the last OCIO working group, some mentioned to avoid having camera related code (i.e. vendor specific code) in the core library. However, it added unnecessary complexity to handle 'normal' built-in transforms versus 'supplemental' built-in transforms (i.e. camera ones) when both are needed most of the time. So, we moved it in the core library with a compilation flag (i.e. could be removed on demand). But the compilation flag in ON by default.

Reply:
The option being default _should_ prevent most issues, I just fear that someone will turn it off and suddenly a config will lack support in that application. What would you think about including a warning if it is off, informing the builder that will introduce potential compatibility issues with some configs?

Reply:
Good idea.

Reply:
Done